libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/SignupEmailFragment.java

```diff
@@ -143,16 +143,24 @@ public void onAttach(Context context) {
         super.onAttach(context);
     }
 
-    @Override
-    public void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
+    @Override public void onStart() {
+        super.onStart();
         mGoogleApiClient = new GoogleApiClient.Builder(getActivity())
                 .addConnectionCallbacks(SignupEmailFragment.this)
                 .enableAutoManage(getActivity(), GOOGLE_API_CLIENT_ID, SignupEmailFragment.this)
                 .addApi(Auth.CREDENTIALS_API)
                 .build();
     }
 
+    @Override
+    public void onStop() {
+        super.onStop();
+        mGoogleApiClient.stopAutoManage(getActivity());
+        if (mGoogleApiClient != null && mGoogleApiClient.isConnected()) {
```

Comment:
Maybe we can move the null check on `mGoogleApiClient` a level above so to protect also the access to `stopAutoManage`. wdyt?

Reply:
you're right. I've originally had the full check there but it was crashing. In this case we should split the check.